### DISCUSSION TITLE
Where is the Chat box?

### DISCUSSION THREAD

**Discussion by devonuto** [community/community#51801] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I watched the video of the demo, when the demo showed when you right-click on the code page, there is an option at the top to ask Copilot a question, but I do not have that option.  

I can see the icon down the bottom left showing it's enabled globally etc, but I cannot ask questions.  

Do you know if this has not been rolled out yet?  The only thing I get is the window where it just randomly suggests garbage code.

**Comment by subeom7** [comment, marked as answer]
I believe that the feature is only for Copilot X and hasn't been rolled out to all users yet. I recommend keeping an eye on the GitHub Copilot release note and the documentation page for updates and announcements. 

Hope this helps!

**Reply by devonuto** [reply]
Okay thanks.  Probably should have waited to get it until it's actually feature ready.  What is there now isn't very useful.

**Comment by albero94** [comment]
I joined the chat waitlist and I now I see
> You're in!
The GitHub Copilot chat private beta is already enabled on your account!

But there is no chat icon in VS Code....

**Reply by ChrisPei** [reply]
same

**Reply by cbarber713** [reply]
> I joined the chat waitlist and I now I see
> 
> > You're in!
> > The GitHub Copilot chat private beta is already enabled on your account!
> 
> But there is no chat icon in VS Code....

Did you download the special version of VSCode that is required for the chat to work? This is all in the welcome email you received when you gained access to the beta. Give it a read and follow the instructions.

**Reply by albero94** [reply]
Do you have a copy of those instructions? I don't see any email.

**Reply by cbarber713** [reply]
Here is a copy of the full email:

You're off the waitlist and can start using Copilot chat. 
Getting Started
Here are the instructions to get started:
Before starting, please make sure that the GitHub ID you intend to use has an active subscription to GitHub Copilot and matches the ID that you used to join the GitHub Copilot chat waitlist.
For Visual Studio Code:
1.	Download and install [Visual Studio Code - Insiders](https://code.visualstudio.com/insiders/)
Already on the Insiders build? Make sure you are on the latest version! 
2.	In VS Code Insiders, go to the Extensions tab (Cmd+Shift+X) in the activity panel and search for and install the [GitHub Copilot Nightly extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly).
If you have not previously authorized VS Code Insiders in your GitHub account, follow the steps to do so, you will be prompted to sign in. 
3.	A new Chat icon appears in the activity panel. Click on it to see the conversation interface.
If that doesn't work, restart the editor and authorize GitHub Copilot to see the chat window.
For Visual Studio:
1.	Ensure you're running Visual Studio 2022 17.5.1 or higher. If not, please update. 
2.	Download and install the [Visual Studio extension for GitHub Copilot - Preview](https://aka.ms/VSXGHCopilot) 
3.	Ensure you are logged in to Visual Studio using the same GitHub ID that you signed up for the waitlist using. If you are not signed with an authorized account you’ll be prompted to do so. If you are already logged in with your GitHub ID in your Visual Studio, please check you don’t need to re-enter your GitHub credentials. Here is the link with instructions to [Add your GitHub ID to your Visual Studio signin](https://learn.microsoft.com/visualstudio/ide/work-with-github-accounts?view=vs-2022).
4.	Start Visual Studio. Once fully loaded, check that the GitHub Copilot chat window shows on your View menu. Then verify that “Ask Copilot” appears at the top of the right click menu in your code editor when a code file is open.
Feedback and Issues
This is a technical preview release so you are likely to encounter some bugs and issues. Please send [feedback](https://aka.ms/microsoft/vscode-copilot-release) while you're using the technical preview. 
For Copilot Chat in VS Code, use the feedback button in the chat panel. 
For Visual Studio, please [use this survey](https://aka.ms/VisualStudio.GitHub.Copilot/feedback) to raise any bugs and issues you find. 
Terms of Use
When you install the extension, you agree to [GitHub Copilot chat preview terms](https://docs.github.com/en/early-access/copilot/github-copilot-chat-technical-preview-license-terms). You should also review this important [transparency](https://docs.github.com/en/early-access/copilot/github-copilot-chat-transparency-note) information about GitHub Copilot Chat preview. 
Thanks,
The GitHub Copilot Team
